package-lock.json

```diff
@@ -47,7 +47,7 @@
         "javascript-stringify": "^2.1.0",
         "js-cookie": "^3.0.1",
         "js-yaml": "^4.1.0",
```

Comment:
>         "liquidjs": "10.0.0",
         "lodash": "^4.17.21",
         "lodash-es": "^4.17.21",
         "lunr": "^2.3.9",
        "lunr-languages": "^1.9.0",
        "mdast-util-from-markdown": "^1.2.0",
        "mdast-util-to-string": "^3.1.0",
        "morgan": "^1.10.0",
        "msgpack5rpc": "^1.1.0",
        "next": "12.2.4",
        "parse5": "7.1.1",
        "port-used": "^2.0.8",
        "react": "^17.0.2",
        "react-dom": "^17.0.2",
        "react-markdown": "^8.0.3",
        "react-syntax-highlighter": "^15.5.0",
        "rehype-autolink-headings": "^6.1.1",
        "rehype-highlight": "^6.0.0",
        "rehype-raw": "^6.1.1",
        "rehype-slug": "^5.0.1",
        "rehype-stringify": "^9.0.3",
        "remark-code-extra": "^1.0.1",
        "remark-gemoji-to-emoji": "^1.1.0",
        "remark-gfm": "^3.0.1",
        "remark-parse": "^10.0.1",
        "remark-parse-no-trim": "^8.0.4",
        "remark-rehype": "^10.1.0",
        "revalidator": "^0.3.1",
        "rss-parser": "^3.12.0",
        "scroll-anchoring": "^0.1.0",
        "semver": "^7.3.7",
        "slash": "^5.0.0",
        "strip-html-comments": "^1.0.0",
        "styled-components": "^5.3.5",
        "swr": "^1.3.0",
        "ts-dedent": "^2.2.0",
        "unified": "^10.1.2",
        "unist-util-visit": "^4.1.0",
        "url-template": "^3.0.0",
        "uuid": "^9.0.0",
        "walk-sync": "^3.0.0"
      },
      "devDependencies": {
        "@actions/core": "^1.9.1",
        "@actions/github": "^5.0.3",
        "@alex_neo/jest-expect-message": "^1.0.5",
        "@babel/core": "^7.18.5",
        "@babel/eslint-parser": "^7.18.2",
        "@babel/plugin-syntax-top-level-await": "^7.14.5",
        "@babel/plugin-transform-runtime": "^7.18.5",
        "@babel/preset-env": "^7.18.2",
        "@graphql-inspector/core": "^3.1.2",
        "@graphql-tools/load": "^7.5.14",
        "@jest/globals": "29.2.2",
        "@octokit/graphql": "5.0.4",
        "@octokit/rest": "^19.0.4",
        "@types/github-slugger": "^2.0.0",
        "@types/imurmurhash": "^0.1.1",
        "@types/js-cookie": "^3.0.2",
        "@types/lodash": "^4.14.182",
        "@types/react": "17.0.38",
        "@types/react-dom": "^18.0.5",
        "@types/react-syntax-highlighter": "^15.5.2",
        "@types/uuid": "^8.3.4",
        "@typescript-eslint/eslint-plugin": "5.42.0",
        "@typescript-eslint/parser": "5.42.0",
        "babel-loader": "^9.0.1",
        "babel-plugin-styled-components": "^2.0.7",
        "babel-preset-env": "^1.7.0",
        "chalk": "^5.0.1",
        "change-case": "^4.1.2",
        "commander": "^9.3.0",
        "cross-env": "^7.0.3",
        "csp-parse": "0.0.2",
        "dedent": "^0.7.0",
        "eslint": "8.26.0",
        "eslint-config-prettier": "^8.5.0",
        "eslint-config-standard": "^17.0.0",
        "eslint-plugin-import": "^2.26.0",
        "eslint-plugin-jsx-a11y": "^6.5.1",
        "eslint-plugin-node": "^11.1.0",
        "eslint-plugin-promise": "^6.0.0",
        "event-to-promise": "^0.8.0",
        "git-diff": "^2.0.6",
        "glob": "^8.0.3",
        "graphql": "^16.5.0",
        "http-status-code": "^2.1.0",
        "husky": "^8.0.1",
        "japanese-characters": "^1.1.0",
        "jest": "29.2.2",
        "jest-environment-puppeteer": "^5.0.4",
        "jest-fail-on-console": "^3.0.1",
        "jest-github-actions-reporter": "^1.0.3",
        "jest-slow-test-reporter": "^1.0.0",
        "kill-port": "2.0.0",
        "linkinator": "4.0.2",
        "lint-staged": "^13.0.1",
        "make-promises-safe": "^5.1.0",
        "minimatch": "^5.1.0",
        "mkdirp": "^1.0.4",
        "mockdate": "^3.0.5",
        "nock": "^13.2.7",
        "nodemon": "2.0.20",
        "npm-merge-driver-install": "^3.0.0",
        "nth-check": "2.1.1",
        "postcss": "^8.4.14",
        "prettier": "^2.7.0",
        "replace": "^1.2.1",
        "rimraf": "^3.0.2",
        "robots-parser": "^3.0.0",
        "sass": "^1.52.3",
        "start-server-and-test": "^1.14.0",
        "strip-ansi": "^7.0.1",
        "typescript": "^4.7.3"
      },
      "engines": {
        "node": ">=16.x"
      },
      "optionalDependencies": {
        "esm": "^3.2.25",
        "image-size": "^1.0.1",
        "jest-puppeteer": "^5.0.4",
        "puppeteer": "^9.1.1",
        "website-scraper": "^5.0.0"
      }
    },
    "node_modules/@actions/core": {
      "version": "1.9.1",
      "resolved": "https://registry.npmjs.org/@actions/core/-/core-1.9.1.tgz",
      "integrity": "sha512-5ad+U2YGrmmiw6du20AQW5XuWo7UKN2052FjSV7MX+Wfjf8sCqcsZe62NfgHys4QI4/Y+vQvLKYL8jWtA1ZBTA==",
      "dev": true,
      "dependencies": {
        "@actions/http-client": "^2.0.1",
        "uuid": "^8.3.2"
      }
    },
    "node_modules/@actions/core/node_modules/uuid": {
      "version": "8.3.2",
      "resolved": "https://registry.npmjs.org/uuid/-/uuid-8.3.2.tgz",
      "integrity": "sha512-+NYs2QeMWy+GWFOEm9xnn6HCDp0l7QBD7ml8zLUmJ+93Q5NF0NocErnwkTkXVFNiX3/fpC6afS8Dhb/gz7R7eg==",
      "dev": true,
      "bin": {
        "uuid": "dist/bin/uuid"
      }
    },
    "node_modules/@actions/github": {
      "version": "5.0.3",
      "resolved": "https://registry.npmjs.org/@actions/github/-/github-5.0.3.tgz",
      "integrity": "sha512-myjA/pdLQfhUGLtRZC/J4L1RXOG4o6aYdiEq+zr5wVVKljzbFld+xv10k1FX6IkIJtNxbAq44BdwSNpQ015P0A==",
      "dev": true,
      "dependencies": {
        "@actions/http-client": "^2.0.1",
        "@octokit/core": "^3.6.0",
        "@octokit/plugin-paginate-rest": "^2.17.0",
        "@octokit/plugin-rest-endpoint-methods": "^5.13.0"
      }
    },
    "node_modules/@actions/http-client": {
      "version": "2.0.1",
      "resolved": "https://registry.npmjs.org/@actions/http-client/-/http-client-2.0.1.tgz",
      "integrity": "sha512-PIXiMVtz6VvyaRsGY268qvj57hXQEpsYogYOu2nrQhlf+XCGmZstmuZBbAybUl1nQGnvS1k1eEsQ69ZoD7xlSw==",
      "dev": true,
      "dependencies": {
        "tunnel": "^0.0.6"
      }
    },
    "node_modules/@alex_neo/jest-expect-message": {
      "version": "1.0.5",
      "dev": true,
      "license": "MIT"
    },
    "node_modules/@ampproject/remapping": {
      "version": "2.2.0",
      "resolved": "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.2.0.tgz",
      "integrity": "sha512-qRmjj8nj9qmLTQXXmaR1cck3UXSRMPrbsLJAasZpF+t3riI71BXed5ebIOYwQntykeZuhjsdweEc9BxH5Jc26w==",
      "dev": true,
      "dependencies": {
        "@jridgewell/gen-mapping": "^0.1.0",
        "@jridgewell/trace-mapping": "^0.3.9"
      },
      "engines": {
        "node": ">=6.0.0"
      }
    },
    "node_modules/@babel/code-frame": {
      "version": "7.16.7",
      "license": "MIT",
      "dependencies": {
        "@babel/highlight": "^7.16.7"
      },
      "engines": {
        "node": ">=6.9.0"
      }
    },
    "node_modules/@babel/compat-data": {
      "version": "7.18.5",
      "resolved": "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.18.5.tgz",
      "integrity": "sha512-BxhE40PVCBxVEJsSBhB6UWyAuqJRxGsAw8BdHMJ3AKGydcwuWW4kOO3HmqBQAdcq/OP+/DlTVxLvsCzRTnZuGg==",
      "dev": true,
      "engines": {
        "node": ">=6.9.0"
      }
    },
    "node_modules/@babel/core": {
      "version": "7.18.5",
      "resolved": "https://registry.npmjs.org/@babel/core/-/core-7.18.5.tgz",
      "integrity": "sha512-MGY8vg3DxMnctw0LdvSEojOsumc70g0t18gNyUdAZqB1Rpd1Bqo/svHGvt+UJ6JcGX+DIekGFDxxIWofBxLCnQ==",
      "dev": true,
      "dependencies": {
        "@ampproject/remapping": "^2.1.0",
        "@babel/code-frame": "^7.16.7",
        "@babel/generator": "^7.18.2",
        "@babel/helper-compilation-targets": "^7.18.2",
        "@babel/helper-module-transforms": "^7.18.0",
        "@babel/helpers": "^7.18.2",
        "@babel/parser": "^7.18.5",
        "@babel/template": "^7.16.7",
        "@babel/traverse": "^7.18.5",
        "@babel/types": "^7.18.4",
        "convert-source-map": "^1.7.0",
        "debug": "^4.1.0",
        "gensync": "^1.0.0-beta.2",
        "json5": "^2.2.1",
        "semver": "^6.3.0"
      },
      "engines": {
        "node": ">=6.9.0"
      },
      "funding": {
        "type": "opencollective",
        "url": "https://opencollective.com/babel"
      }
    },
    "node_modules/@babel/core/node_modules/semver": {
      "version": "6.3.0",
      "dev": true,
      "license": "ISC",
      "bin": {
        "semver": "bin/semver.js"
      }
    },
    "node_modules/@babel/eslint-parser": {
      "version": "7.18.2",
      "resolved": "https://registry.npmjs.org/@babel/eslint-parser/-/eslint-parser-7.18.2.tgz",
      "integrity": "sha512-oFQYkE8SuH14+uR51JVAmdqwKYXGRjEXx7s+WiagVjqQ+HPE+nnwyF2qlVG8evUsUHmPcA+6YXMEDbIhEyQc5A==",
      "dev": true,
      "dependencies": {
        "eslint-scope": "^5.1.1",
        "eslint-visitor-keys": "^2.1.0",
        "semver": "^6.3.0"
      },
      "engines": {
        "node": "^10.13.0 || ^12.13.0 || >=14.0.0"
      },
      "peerDependencies": {
        "@babel/core": ">=7.11.0",
        "eslint": "^7.5.0 || ^8.0.0"
      }
    },
    "node_modules/@babel/eslint-parser/node_modules/semver": {
      "version": "6.3.0",
      "dev": true,
      "license": "ISC",
      "bin": {
 @@ -13622,434 +13622,435 @@
    "node_modules/js-cookie": {
      "version": "3.0.1",
      "license": "MIT",
      "engines": {
        "node": ">=12"
      }
    },
    "node_modules/js-sdsl": {
      "version": "4.1.4",
      "resolved": "https://registry.npmjs.org/js-sdsl/-/js-sdsl-4.1.4.tgz",
      "integrity": "sha512-Y2/yD55y5jteOAmY50JbUZYwk3CP3wnLPEZnlR1w9oKhITrBEtAxwuWKebFf8hMrPMgbYwFoWK/lH2sBkErELw==",
      "dev": true
    },
    "node_modules/js-tokens": {
      "version": "4.0.0",
      "license": "MIT"
    },
    "node_modules/js-yaml": {
      "version": "4.1.0",
      "license": "MIT",
      "dependencies": {
        "argparse": "^2.0.1"
      },
      "bin": {
        "js-yaml": "bin/js-yaml.js"
      }
    },
    "node_modules/jsesc": {
      "version": "2.5.2",
      "license": "MIT",
      "bin": {
        "jsesc": "bin/jsesc"
      },
      "engines": {
        "node": ">=4"
      }
    },
    "node_modules/json-buffer": {
      "version": "3.0.1",
      "license": "MIT"
    },
    "node_modules/json-parse-even-better-errors": {
      "version": "2.3.1",
      "dev": true,
      "license": "MIT"
    },
    "node_modules/json-schema-traverse": {
      "version": "1.0.0",
      "license": "MIT"
    },
    "node_modules/json-stable-stringify-without-jsonify": {
      "version": "1.0.1",
      "dev": true,
      "license": "MIT"
    },
    "node_modules/json-stringify-safe": {
      "version": "5.0.1",
      "dev": true,
      "license": "ISC"
    },
    "node_modules/json5": {
      "version": "2.2.1",
      "resolved": "https://registry.npmjs.org/json5/-/json5-2.2.1.tgz",
      "integrity": "sha512-1hqLFMSrGHRHxav9q9gNjJ5EXznIxGVO09xQRrwplcS8qs28pZ8s8hupZAmqDwZUmVZ2Qb2jnyPOWcDH8m8dlA==",
      "dev": true,
      "bin": {
        "json5": "lib/cli.js"
      },
      "engines": {
        "node": ">=6"
      }
    },
    "node_modules/jsonfile": {
      "version": "6.1.0",
      "resolved": "https://registry.npmjs.org/jsonfile/-/jsonfile-6.1.0.tgz",
      "integrity": "sha512-5dgndWOriYSm5cnYaJNhalLNDKOqFwyDB/rr1E9ZsGciGvKPs8R2xYGCacuf3z6K1YKDz182fd+fY3cn3pMqXQ==",
      "optional": true,
      "dependencies": {
        "universalify": "^2.0.0"
      },
      "optionalDependencies": {
        "graceful-fs": "^4.1.6"
      }
    },
    "node_modules/jsx-ast-utils": {
      "version": "3.2.1",
      "dev": true,
      "license": "MIT",
      "dependencies": {
        "array-includes": "^3.1.3",
        "object.assign": "^4.1.2"
      },
      "engines": {
        "node": ">=4.0"
      }
    },
    "node_modules/keyv": {
      "version": "4.0.3",
      "license": "MIT",
      "dependencies": {
        "json-buffer": "3.0.1"
      }
    },
    "node_modules/kill-port": {
      "version": "2.0.0",
      "dev": true,
      "license": "MIT",
      "dependencies": {
        "get-them-args": "1.3.2",
        "shell-exec": "1.0.2"
      },
      "bin": {
        "kill-port": "cli.js"
      }
    },
    "node_modules/kind-of": {
      "version": "6.0.3",
      "license": "MIT",
      "engines": {
        "node": ">=0.10.0"
      }
    },
    "node_modules/kleur": {
      "version": "4.1.4",
      "license": "MIT",
      "engines": {
        "node": ">=6"
      }
    },
    "node_modules/language-subtag-registry": {
      "version": "0.3.21",
      "dev": true,
      "license": "ODC-By-1.0"
    },
    "node_modules/language-tags": {
      "version": "1.0.5",
      "dev": true,
      "license": "MIT",
      "dependencies": {
        "language-subtag-registry": "~0.3.2"
      }
    },
    "node_modules/latest-version": {
      "version": "7.0.0",
      "resolved": "https://registry.npmjs.org/latest-version/-/latest-version-7.0.0.tgz",
      "integrity": "sha512-KvNT4XqAMzdcL6ka6Tl3i2lYeFDgXNCuIX+xNx6ZMVR1dFq+idXd9FLKNMOIx0t9mJ9/HudyX4oZWXZQ0UJHeg==",
      "dev": true,
      "dependencies": {
        "package-json": "^8.1.0"
      },
      "engines": {
        "node": ">=14.16"
      },
      "funding": {
        "url": "https://github.com/sponsors/sindresorhus"
      }
    },
    "node_modules/lazy-ass": {
      "version": "1.6.0",
      "dev": true,
      "license": "MIT",
      "engines": {
        "node": "> 0.8"
      }
    },
    "node_modules/lazy-cache": {
      "version": "1.0.4",
      "devOptional": true,
      "license": "MIT",
      "engines": {
        "node": ">=0.10.0"
      }
    },
    "node_modules/leven": {
      "version": "3.1.0",
      "resolved": "https://registry.npmjs.org/leven/-/leven-3.1.0.tgz",
      "integrity": "sha512-qsda+H8jTaUaN/x5vzW2rzc+8Rw4TAQ/4KjB46IwK5VH+IlVeeeje/EoZRpiXvIqjFgK84QffqPztGI3VBLG1A==",
      "dev": true,
      "engines": {
        "node": ">=6"
      }
    },
    "node_modules/levn": {
      "version": "0.4.1",
      "dev": true,
      "license": "MIT",
      "dependencies": {
        "prelude-ls": "^1.2.1",
        "type-check": "~0.4.0"
      },
      "engines": {
        "node": ">= 0.8.0"
      }
    },
    "node_modules/lilconfig": {
      "version": "2.0.5",
      "resolved": "https://registry.npmjs.org/lilconfig/-/lilconfig-2.0.5.tgz",
      "integrity": "sha512-xaYmXZtTHPAw5m+xLN8ab9C+3a8YmV3asNSPOATITbtwrfbwaLJj8h66H1WMIpALCkqsIzK3h7oQ+PdX+LQ9Eg==",
      "dev": true,
      "engines": {
        "node": ">=10"
      }
    },
    "node_modules/lines-and-columns": {
      "version": "1.2.4",
      "dev": true,
      "license": "MIT"
    },
    "node_modules/linkinator": {
      "version": "4.0.2",
      "resolved": "https://registry.npmjs.org/linkinator/-/linkinator-4.0.2.tgz",
      "integrity": "sha512-PkwPiMx2NsRXaKQiOkTH8uMkkUrN3rh0gI6nFwjme2A7TVVYzlwGfmb00wbDEEdLrO49tDbajUEJGup64swmog==",
      "dev": true,
      "dependencies": {
        "chalk": "^5.0.0",
        "escape-html": "^1.0.3",
        "gaxios": "^5.0.0",
        "glob": "^8.0.1",
        "htmlparser2": "^8.0.1",
        "marked": "^4.0.3",
        "meow": "^10.1.1",
        "mime": "^3.0.0",
        "server-destroy": "^1.0.1",
        "srcset": "^5.0.0",
        "update-notifier": "^6.0.0"
      },
      "bin": {
        "linkinator": "build/src/cli.js"
      },
      "engines": {
        "node": ">=14"
      }
    },
    "node_modules/linkinator/node_modules/mime": {
      "version": "3.0.0",
      "dev": true,
      "license": "MIT",
      "bin": {
        "mime": "cli.js"
      },
      "engines": {
        "node": ">=10.0.0"
      }
    },
    "node_modules/lint-staged": {
      "version": "13.0.3",
      "resolved": "https://registry.npmjs.org/lint-staged/-/lint-staged-13.0.3.tgz",
      "integrity": "sha512-9hmrwSCFroTSYLjflGI8Uk+GWAwMB4OlpU4bMJEAT5d/llQwtYKoim4bLOyLCuWFAhWEupE0vkIFqtw/WIsPug==",
      "dev": true,
      "dependencies": {
        "cli-truncate": "^3.1.0",
        "colorette": "^2.0.17",
        "commander": "^9.3.0",
        "debug": "^4.3.4",
        "execa": "^6.1.0",
        "lilconfig": "2.0.5",
        "listr2": "^4.0.5",
        "micromatch": "^4.0.5",
        "normalize-path": "^3.0.0",
        "object-inspect": "^1.12.2",
        "pidtree": "^0.6.0",
        "string-argv": "^0.3.1",
        "yaml": "^2.1.1"
      },
      "bin": {
        "lint-staged": "bin/lint-staged.js"
      },
      "engines": {
        "node": "^14.13.1 || >=16.0.0"
      },
      "funding": {
        "url": "https://opencollective.com/lint-staged"
      }
    },
    "node_modules/lint-staged/node_modules/debug": {
      "version": "4.3.4",
      "resolved": "https://registry.npmjs.org/debug/-/debug-4.3.4.tgz",
      "integrity": "sha512-PRWFHuSU3eDtQJPvnNY7Jcket1j0t5OuOsFzPPzsekD52Zl8qUfFIPEiswXqIvHWGVHOgX+7G/vCNNhehwxfkQ==",
      "dev": true,
      "dependencies": {
        "ms": "2.1.2"
      },
      "engines": {
        "node": ">=6.0"
      },
      "peerDependenciesMeta": {
        "supports-color": {
          "optional": true
        }
      }
    },
    "node_modules/lint-staged/node_modules/execa": {
      "version": "6.1.0",
      "resolved": "https://registry.npmjs.org/execa/-/execa-6.1.0.tgz",
      "integrity": "sha512-QVWlX2e50heYJcCPG0iWtf8r0xjEYfz/OYLGDYH+IyjWezzPNxz63qNFOu0l4YftGWuizFVZHHs8PrLU5p2IDA==",
      "dev": true,
      "dependencies": {
        "cross-spawn": "^7.0.3",
        "get-stream": "^6.0.1",
        "human-signals": "^3.0.1",
        "is-stream": "^3.0.0",
        "merge-stream": "^2.0.0",
        "npm-run-path": "^5.1.0",
        "onetime": "^6.0.0",
        "signal-exit": "^3.0.7",
        "strip-final-newline": "^3.0.0"
      },
      "engines": {
        "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
      },
      "funding": {
        "url": "https://github.com/sindresorhus/execa?sponsor=1"
      }
    },
    "node_modules/lint-staged/node_modules/get-stream": {
      "version": "6.0.1",
      "resolved": "https://registry.npmjs.org/get-stream/-/get-stream-6.0.1.tgz",
      "integrity": "sha512-ts6Wi+2j3jQjqi70w5AlN8DFnkSwC+MqmxEzdEALB2qXZYV3X/b1CTfgPLGJNMeAWxdPfU8FO1ms3NUfaHCPYg==",
      "dev": true,
      "engines": {
        "node": ">=10"
      },
      "funding": {
        "url": "https://github.com/sponsors/sindresorhus"
      }
    },
    "node_modules/lint-staged/node_modules/human-signals": {
      "version": "3.0.1",
      "resolved": "https://registry.npmjs.org/human-signals/-/human-signals-3.0.1.tgz",
      "integrity": "sha512-rQLskxnM/5OCldHo+wNXbpVgDn5A17CUoKX+7Sokwaknlq7CdSnphy0W39GU8dw59XiCXmFXDg4fRuckQRKewQ==",
      "dev": true,
      "engines": {
        "node": ">=12.20.0"
      }
    },
    "node_modules/lint-staged/node_modules/is-stream": {
      "version": "3.0.0",
      "resolved": "https://registry.npmjs.org/is-stream/-/is-stream-3.0.0.tgz",
      "integrity": "sha512-LnQR4bZ9IADDRSkvpqMGvt/tEJWclzklNgSw48V5EAaAeDd6qGvN8ei6k5p0tvxSR171VmGyHuTiAOfxAbr8kA==",
      "dev": true,
      "engines": {
        "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
      },
      "funding": {
        "url": "https://github.com/sponsors/sindresorhus"
      }
    },
    "node_modules/lint-staged/node_modules/mimic-fn": {
      "version": "4.0.0",
      "resolved": "https://registry.npmjs.org/mimic-fn/-/mimic-fn-4.0.0.tgz",
      "integrity": "sha512-vqiC06CuhBTUdZH+RYl8sFrL096vA45Ok5ISO6sE/Mr1jRbGH4Csnhi8f3wKVl7x8mO4Au7Ir9D3Oyv1VYMFJw==",
      "dev": true,
      "engines": {
        "node": ">=12"
      },
      "funding": {
        "url": "https://github.com/sponsors/sindresorhus"
      }
    },
    "node_modules/lint-staged/node_modules/npm-run-path": {
      "version": "5.1.0",
      "resolved": "https://registry.npmjs.org/npm-run-path/-/npm-run-path-5.1.0.tgz",
      "integrity": "sha512-sJOdmRGrY2sjNTRMbSvluQqg+8X7ZK61yvzBEIDhz4f8z1TZFYABsqjjCBd/0PUNE9M6QDgHJXQkGUEm7Q+l9Q==",
      "dev": true,
      "dependencies": {
        "path-key": "^4.0.0"
      },
      "engines": {
        "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
      },
      "funding": {
        "url": "https://github.com/sponsors/sindresorhus"
      }
    },
    "node_modules/lint-staged/node_modules/object-inspect": {
      "version": "1.12.2",
      "resolved": "https://registry.npmjs.org/object-inspect/-/object-inspect-1.12.2.tgz",
      "integrity": "sha512-z+cPxW0QGUp0mcqcsgQyLVRDoXFQbXOwBaqyF7VIgI4TWNQsDHrBpUQslRmIfAoYWdYzs6UlKJtB2XJpTaNSpQ==",
      "dev": true,
      "funding": {
        "url": "https://github.com/sponsors/ljharb"
      }
    },
    "node_modules/lint-staged/node_modules/onetime": {
      "version": "6.0.0",
      "resolved": "https://registry.npmjs.org/onetime/-/onetime-6.0.0.tgz",
      "integrity": "sha512-1FlR+gjXK7X+AsAHso35MnyN5KqGwJRi/31ft6x0M194ht7S+rWAvd7PHss9xSKMzE0asv1pyIHaJYq+BbacAQ==",
      "dev": true,
      "dependencies": {
        "mimic-fn": "^4.0.0"
      },
      "engines": {
        "node": ">=12"
      },
      "funding": {
        "url": "https://github.com/sponsors/sindresorhus"
      }
    },
    "node_modules/lint-staged/node_modules/path-key": {
      "version": "4.0.0",
      "resolved": "https://registry.npmjs.org/path-key/-/path-key-4.0.0.tgz",
      "integrity": "sha512-haREypq7xkM7ErfgIyA0z+Bj4AGKlMSdlQE2jvJo6huWD1EdkKYV+G/T4nq0YEF2vgTT8kqMFKo1uHn950r4SQ==",
      "dev": true,
      "engines": {
        "node": ">=12"
      },
      "funding": {
        "url": "https://github.com/sponsors/sindresorhus"
      }
    },
    "node_modules/lint-staged/node_modules/strip-final-newline": {
      "version": "3.0.0",
      "resolved": "https://registry.npmjs.org/strip-final-newline/-/strip-final-newline-3.0.0.tgz",
      "integrity": "sha512-dOESqjYr96iWYylGObzd39EuNTa5VJxyvVAEm5Jnh7KGo75V43Hk1odPQkNDyXNmUR6k+gEiDVXnjB8HJ3crXw==",
      "dev": true,
      "engines": {
        "node": ">=12"
      },
      "funding": {
        "url": "https://github.com/sponsors/sindresorhus"
       }
     },
     "node_modules/liquidjs": {